dbtransfer.py

```diff
@@ -142,13 +142,17 @@ def upload( self, oldkey, *args, **kwargs ):
 					"parentdir": "",
 					"parentrepo": "",
 					"weak": False } )
+
+			oldkey = conf["viur.mainApp"].file.decodeFileName(oldkey)
```

Comment:
Suggestion: "self" is not used in decodeFileName. We could make this function a class method and call it directly (instead of relying that a file module is loaded under conf["viur.mainApp"].file)